types/twilio-video/index.d.ts

```diff
@@ -583,6 +584,9 @@ export interface LogLevels {
     signaling: LogLevel;
     webrtc: LogLevel;
 }
+export interface MediaStreamTrackPublishOptions extends LocalTrackOptions {
+    priority: Track.Priority;
```

Comment:
```suggestion
    priority?: Track.Priority;
```
should be probably optional, as it defaults to `standard`, am I correct?
https://media.twiliocdn.com/sdk/js/video/releases/2.0.0/docs/global.html#MediaStreamTrackPublishOptions

Reply:
You are totally right, fixed in bf4abac7e9af18db682a327883398c558037bd18